### DISCUSSION TITLE
Update 1.22 not found

### DISCUSSION THREAD

**Discussion by metbril** [FreshRSS/FreshRSS#5751] [Q&A] [answered]
I have updated over the web a few times without issues, but the last week(s) I am unable to upgrade from 1.21 to 1.22. The update pages says 'no updates found'.

Is this a know issue?

Any help is appreciated.

![image](https://github.com/FreshRSS/FreshRSS/assets/559346/14441717-98a5-4dcc-97d6-d485d2f6159f)

**Comment by Alkarex** [comment]
The built-in ZIP update always comes last. We will shortly release 1.22.1 with a few bug fixes, and then it will appear there as well

**Comment by daffydock** [comment]
Sounds great. I will wait. Would you be so kind as to update this Issue as to when it is updated? 

For those who are curious, you can always check for yourself if it has been updated here:
https://github.com/FreshRSS/update.freshrss.org

**Comment by marienfressinaud** [comment, marked as answer]
@daffydock @metbril You should be able to update to FreshRSS 1.22.1 now

**Reply by beerisgood** [reply]
Can confirm that using web update works without problems to 1.22.1 👍

**Comment by metbril** [comment]
Everything went smoothly

![image](https://github.com/FreshRSS/FreshRSS/assets/559346/6396bbc6-9b74-443b-8cf0-8cd8df7c91c5)
